tdrs-backend/tdpservice/reports/apps.py

```diff
@@ -5,5 +5,5 @@
 class ReportsConfig(AppConfig):
     """Define Report properties."""
 
-    name = "reports"
+    name = "tdpservice.reports"
```

Comment:
This should always be the full python path per the [docs](https://docs.djangoproject.com/en/2.2/ref/applications/#django.apps.AppConfig.name)

Since a new package was installed with this PR a minor version update of Django was applied per our configuration. This version of Django introduces [automatic app config discovery](https://docs.djangoproject.com/en/3.2/releases/3.2/#automatic-appconfig-discovery) and now throws an error preventing startup if these names are not defined as full paths:
```
web_1          | django.core.exceptions.ImproperlyConfigured: Cannot import 'reports'. Check that 'tdpservice.reports.apps.ReportsConfig.name' is correct.
```